src/main/java/core/basesyntax/exception/InvalidUserDataException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class InvalidUserDataException extends Exception {
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/225599368-83bf3101-b7da-49de-8605-c9675d866c9a.png)
```suggestion
public class RegistrationException extends RuntimeException {
```